### PR TITLE
Align feature page icons with feature list

### PR DESCRIPTION
**Pull request by clelland**

### CODE REVIEW

**Comment by clelland** [comment]
This fixes #331

File: templates/features.html [hunk 1]
```diff
@@ -135,7 +135,7 @@
 
 // Fire of features.json XHR right away so data can populate faster.
 var featuresPromise = new Promise(function(resolve, reject) {
-  var url = location.hostname == 'localhost' ? 'https://www.chromestatus.com/features.json' :
+  var url = location.hostname != 'localhost' ? 'https://www.chromestatus.com/features.json' :
```
Comment (RByers) [inline templates/features.html]: not intended as part of this PR - right?

**Comment by RByers** [comment]
LGTM with above nit

File: models.py [hunk 2]
```diff
@@ -381,6 +383,28 @@ def get_shipping_samples(self, limit=None, update_cache=False):
 
     return feature_list
 
+  def crbug_number(self):
+    if not self.bug_url:
+      return;
```
Comment (ebidel) [inline models.py]: remove ;

File: models.py [hunk 3]
```diff
@@ -381,6 +383,28 @@ def get_shipping_samples(self, limit=None, update_cache=False):
...
+    crbug_number = self.crbug_number()
+    if crbug_number and self.impl_status_chrome in (
+        NO_ACTIVE_DEV,
+        PROPOSED,
+        IN_DEVELOPMENT,
+        BEHIND_A_FLAG,
+        EXPERIMENTAL_FRAMEWORK):
+      params.append('blocking=' + crbug_number);
+    if self.owner:
+      params.append('cc=' + ",".join(self.owner))
```
Comment (ebidel) [inline models.py]: single quotes

File: models.py [hunk 4]
```diff
@@ -381,6 +383,28 @@ def get_shipping_samples(self, limit=None, update_cache=False):
...
+    return url + '?' + "&".join(params)
```
Comment (ebidel) [inline models.py]: single quotes

File: static/sass/features/feature.scss [hunk 5]
```diff
@@ -39,6 +39,18 @@ section {
   }
 }
 
+iron-icon {
```
Comment (ebidel) [inline static/sass/features/feature.scss]: Can you add a comment here DRY this up with the rules in chromedash-feature.scss?

File: templates/feature.html [hunk 6]
```diff
@@ -16,6 +16,10 @@
 <link rel="alternate" type="application/rss+xml" href="http://www.chromestatus.com/features.xml" title="All features" />
 {% endblock %}
 
+{% block html_imports %}
+<link rel="import" href="/static/elements/features-imports{% if VULCANIZE %}.vulcanize{% endif %}.html">
```
Comment (ebidel) [inline templates/feature.html]: In theory, at some point we should be able to just switch this page over to use `<chromedash-feature>`.

File: templates/feature.html [hunk 7]
```diff
@@ -25,7 +29,24 @@
       {% if feature.impl_status_chrome == "Removed" %} (removed){% endif %}
     </h2>
     <a href="/features#category: {{ feature.category }}" class="category">{{ feature.category }}</a>
-    {% if user.is_whitelisted %}<a href="/admin/features/edit/{{ feature.id }}" class="edit">&#160;edit</a>{% endif %}
+    {% if feature.meta.experimentalframework %}&nbsp;
```
Comment (ebidel) [inline templates/feature.html]: Can you define a spacing in CSS rather than &nbsp;?. Same for below.

File: templates/features.html (hunk 1 above)
Comment (clelland) [inline templates/features.html]: !! Definitely not, thanks!

File: models.py (hunk 2 above)
Comment (clelland) [inline models.py]: Done.

File: models.py (hunk 4 above)
Comment (clelland) [inline models.py]: Done

File: templates/feature.html (hunk 7 above)
Comment (clelland) [inline templates/feature.html]: I've copied some of the div structure and CSS from the polymer element into here now, so the spacing is consistent. As you mentioned above, though, this should really be just *using* the `<chromedash-feature>` element. We should factor that out in the future.

File: models.py [hunk 8]
```diff
@@ -381,6 +383,28 @@ def get_shipping_samples(self, limit=None, update_cache=False):
...
+    url = 'https://bugs.chromium.org/p/chromium/issues/entry';
+    params = ['components=Blink'];
+    crbug_number = self.crbug_number()
+    if crbug_number and self.impl_status_chrome in (
+        NO_ACTIVE_DEV,
+        PROPOSED,
+        IN_DEVELOPMENT,
+        BEHIND_A_FLAG,
+        EXPERIMENTAL_FRAMEWORK):
+      params.append('blocking=' + crbug_number);
```
Comment (ebidel) [inline models.py]: remove ";"

**Comment by ebidel** [comment]
LGTM. Just caught another ";" to remove

File: models.py (hunk 8 above)
Comment (clelland) [inline models.py]: Ack! That's what I get for translating from JavaScript by hand. Thanks!

**Pull request merged**